src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/ClientMethodsAnalyzer.cs

```diff
@@ -58,7 +58,7 @@ public override void AnalyzeCore(ISymbolAnalysisContext context)
                         CheckClientMethod(context, syncMember);
                     }
                 }
-            }
-        }
+            }
+        }
```

Comment:
?

Reply:
Seems like some kind of whitespace normalization, checking.